shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -76,18 +88,46 @@ private void chown(AlluxioURI path, String owner, boolean recursive)
     System.out.println("Changed owner of " + path + " to " + owner);
   }
 
+  /**
+   * Changes the owner and group for the path specified in args.
+   *
+   * @param path the {@link AlluxioURI} path to update
+   * @param owner the new owner
+   * @param group the new group
+   * @param recursive whether to change the owner and group recursively
+   */
+  private void chown(AlluxioURI path, String owner, String group, boolean recursive)
+      throws AlluxioException, IOException {
+    SetAttributeOptions options =
+        SetAttributeOptions.defaults().setOwner(owner).setGroup(group).setRecursive(recursive);
+    mFileSystem.setAttribute(path, options);
+    System.out.println("Changed owner:group of " + path + " to " + owner + ":" + group + ".");
+  }
+
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
-    String owner = args[0];
     AlluxioURI path = new AlluxioURI(args[1]);
-    chown(path, owner, cl.hasOption("R"));
-    return 0;
+    Matcher matchUserGroup = USER_GROUP_PATTERN.matcher(args[0]);
+    if (matchUserGroup.matches()) {
+      String owner = matchUserGroup.group("user");
+      String group = matchUserGroup.group("group");
+      chown(path, owner, group, cl.hasOption("R"));
+      return 0;
+    }
+    Matcher matchUserOnly = USER_PATTERN.matcher(args[0]);
+    if (matchUserOnly.matches()) {
+      String owner = matchUserOnly.group("user");
+      chown(path, owner, cl.hasOption("R"));
+      return 0;
+    }
+    System.out.println("Failed to parse " + args[0] + "as user or user:group");
```

Comment:
There is no test coverage for the parsing in the integration test. 
Can you create `shell/src/test/java/alluxio/shell/command/ChownCommandTest.java` as the unit test for this chown command, which focuses on the input validation? 

The difference between unit test and integration test is that unit test focuses on method level, while in integration test, we launch master/worker and test the integration.